src/ai/backend/client/output/console.py

```diff
@@ -117,39 +117,38 @@ def print_paginated_list(
         initial_page_offset: int,
         page_size: int = None,
     ) -> None:
+        fields: Sequence[FieldSpec] = []
+
+        def infinite_fetch(page_size):
```

Comment:
The `page_size` parameter is shadowing definition of already defined `page_size` variable. Let's rename it.
```suggestion
        def infinite_fetch(_page_size):
```